Scripts/PositiveDetectionsVSDetectionEngines/PositiveDetectionsVSDetectionEngines.py

```diff
@@ -4,50 +4,74 @@
 
 
 def extract_engines_data_from_indicator(indicator_data):
-    if not (indicator_data and 'detectionengines' in indicator_data['CustomFields']
-            and 'positivedetections' in indicator_data['CustomFields']):
-        demisto.results("None")
-        sys.exit(0)
+    if not indicator_data:
+        exit_with_message("No indicator found")
+    cstm_fields = indicator_data.get("CustomFields")
+    if not cstm_fields:
+        # No content, so will display 0/0
+        return create_pie(0, 0)
+    elif "detectionengines" not in cstm_fields:
+        exit_with_message('Please provide  Custom Field "Detection Engines"')
+    elif "positivedetections" not in cstm_fields:
+        exit_with_message('Please provide  Custom Field "Positive Detections"')
```

Comment:
where the user will see this messages? `'Please provide  Custom Field "Positive Detections"'`